frontend/src/components/CollabPage/VideoChatPanel.jsx

```diff
@@ -119,7 +119,8 @@ export default function VideoChatPanel() {
 
   const peer = new Peer({
     host: VIDEO_SVC_HOST,
-    port: VIDEO_SVC_PORT,
+    secure: true,
+    port: 443,
```

Comment:
Why does the port have to be 443?

Reply:
Frontend makes https request now, so port 443 here is to ensure https connection can be made between the frontend and server. I will go and try some other configurations.